### ISSUE TITLE
Container Origin 0.5 or 0 

### ISSUE THREAD

**Issue by Kvisaz** [photonstorm/phaser#5604]
## Version

* Phaser Version: 3.53.1

* Operating system: Windows, MacOS

* Browser: Chrome, Safari

## Description
Containers originX and originY setted as 0.5, but in documentation described as 0

![2021-03-16_155436](https://user-images.githubusercontent.com/16399450/111282358-80c84600-8670-11eb-802c-6fe111a53992.png)

![2021-03-16_155906](https://user-images.githubusercontent.com/16399450/111282407-8cb40800-8670-11eb-8c30-e7e8dd8029a3.png)

Also, Phaser.Display.Align.In work incorrectly when container is anchor of aligning

**Comment by samme** [comment]
They are two different origins. Maybe Rich can explain it better.

(0, 0) is the origin for the children's transforms. It's a container-only concept.

(0.5, 0.5) is the container's `position` on its (`width`, `height`). It's used for alignment, placing an input hit area, and placing a physics body.

I think `Align.In()` should work as expected if you use `setSize()` first.

**Comment by photonstorm** [comment]
A Container doesn't have a size, unless you specifically give it one (via `setSize()`). If you do, then this size is used for all alignment calls.

`Container.originX/Y` are fixed at 0.5 and cannot be changed, which is why the docs say:

```
    /**
     * Internal value to allow Containers to be used for input and physics.
     * Do not change this value. It has no effect other than to break things.
     *
     * @name Phaser.GameObjects.Container#originX
     * @type {number}
     * @readonly
     * @override
     * @since 3.4.0
     */
```

The documentation saying that the 'origin' is 0x0 is relating to the transform point around which the Container's children are placed. I have updated the constructor documentation to make this clearer and removed the word 'origin'.
